src/components/playerstats/playerstats.js

```diff
@@ -332,17 +332,17 @@ define(['events', 'globalize', 'playbackManager', 'connectionManager', 'syncplay
         var stats = syncplayManager.getStats();
 
         syncStats.push({
-            label: globalize.translate("LabelSyncplayTimeOffset"),
-            value: stats.TimeOffset + "ms"
+            label: globalize.translate('LabelSyncplayTimeOffset'),
+            value: stats.TimeOffset + globalize.translate('LabelSyncplayMillisecondsUnit')
```

Comment:
Why not just name it `MillisecondsUnit`? I mean, it's highly unlikely milliseconds in other places (not syncplay) would have different localization :)

Also where is the related localization files change?

Reply:
> Also where is the related localization files change?

Should be this one in [`en-us.json`](https://github.com/OancaAndrei/jellyfin-web/commit/84a9e6e15e35d04c3a5607429895342b98c79a18#diff-c3a474b438fc441c65a39d4ce2b1357cR859).

Also, am I supposed to put all new strings in alphabetical order? Or is it going to be fixed automatically?

Reply:
dunno about that, let's ask @dkanada

Reply:
It's not automatic but you don't need to, we can always fix it with some scripts in the root directory if we decide to start sorting them.

Reply:
> Do note that using [date-fns](https://date-fns.org/v2.12.0/docs/) for time (including durations) is preferred to using `globalize` since we're already using it elsewhere in the code and the plan is to replace every instance of dates and times with date-fns.

So, I gave a quick look at `date-dns` and, from what I could see, it doesn't seem to handle milliseconds properly. I tried the following:
```
var helperDate = datefns.addMilliseconds(new Date(0), milliseconds);
var formatted = datefns.format(helperDate, 'TT', {
    locale: dfnshelper.getLocale()
});
```
but the unit is missing.

Were you proposing `date-fns` for formatting only or for operations on dates as well?